app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -315,3 +315,7 @@ a.btn-info span{
   max-height: 80vh;
   overflow-y: auto;
 }
+// Color origin: pf-orange-400
+.pficon-ok.warn:before {
```

Comment:
is this really the right place to put this? wont this change all references to this icon?